pyproject.toml

```diff
@@ -1,6 +1,53 @@
+[project]
+name = "wagtail-treemodeladmin"
+version = "1.5.0"
+description = "TreeModelAdmin for Wagtail"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
The README still specifies [Python 3.6+](https://github.com/cfpb/wagtail-treemodeladmin/blob/main/README.md#dependencies); did you mean to change there too?

Reply:
Yup.

Reply:
Fixed in [acd8e4b](https://github.com/cfpb/wagtail-treemodeladmin/pull/34/commits/acd8e4b6492c440e71a308057b2084ac0f6cb5ff).